src/Traits/SearchFacade.php

```diff
@@ -7,11 +7,12 @@
 
 trait SearchFacade
 {
-    protected function search($query, ?string $index = null, ?int $limit = 10): Collection
+    protected function search($query, string $site, ?string $index = null, ?int $limit = 10): Collection
```

Comment:
Adding site as a second argument will introduce a breaking change. We need to move it to the end and pass null as a default value to avoid that.

Reply:
As well, we need to either pass the $site from the LiveSearch class or fetch the default site right here. 

`->where('site', $site ?? Site::current()->handle)` might already be enough.